### DISCUSSION TITLE
RDF was calculated by shielding three adjacent beads within the same chain

### DISCUSSION THREAD

**Discussion by yingyingwukim** [MDAnalysis/mdanalysis#4711] [Q&A] [answered]
![Uploading 图片1.png…]()

Dear experts：
       A multi-chain polymer system as shown in the figure, how can we use MDAnalysis to calculate the RDF while excluding the nearest 3 atoms within the same chain?
      Here's my code using exclusion_block to exclude nearest neighbors as shown below. Could this approach correctly exclude the nearest 3 atoms within the same chain? Please help improve my code.

```python
# 函数3-计算RDF
def compute_rdf(atom1, atom2):
    """
    计算两个原子组之间的RDF
    """
    rdf = InterRDF(atom1, atom2, nbins=200, range=(0.0,30), exclusion_block=(2, 2))
    rdf.run()
    return rdf.results.bins, rdf.results.rdf
```

**Comment by orbeckst** [comment, marked as answer]
Is this question related to #4716 ?

**Reply by yingyingwukim** [reply]
Yeah, these are all question I sent

**Reply by orbeckst** [reply]
Ok, then I'll close this one; we only need one copy of a question and the other one has more details.
